src/spyglass/common/common_interval.py

```diff
@@ -23,6 +23,7 @@ class IntervalList(SpyglassMixin, dj.Manual):
     interval_list_name: varchar(170)  # descriptive name of this interval list
     ---
     valid_times: longblob  # numpy array with start/end times for each interval
+    interval_type = "": varchar(100)  # type of interval list (e.g. 'position', 'spikesorting_recording_v1')
```

Comment:
This comment leaves room for interpretation I think I would avoid. I don't think we need the `interval_` prefix for the field, which is implied by by the table, but I'm on the fence about using `type` vs `pipeline`
```suggestion
    pipeline = "": varchar(32)  # inserting pipeline (e.g. 'lfp', 'spikesorting_recording_v1')
```

Reply:
I think `type` could be more confusing if someone thinks it means a data type. `pipeline` seems reasonable to me